Makefile

```diff
@@ -5,11 +5,14 @@ admin-build:
 	docker compose run --rm admin bash -c "npm --workspace @socialincome/admin install && npm run admin:build"
 
 admin-serve:
-	docker compose up admin
+	docker compose run --rm admin bash -c "npm --workspace=@socialincome/admin install && npm run admin:serve"
 
 admin-test:
 	docker compose run --rm admin bash -c "npm --workspace @socialincome/admin install && npm run admin:test"
 
+admin-export-seed:
+    docker exec -it public-admin-1 npm run emulators:export
```

Comment:
Are you sure that works in every case with the static public-admin-1 name here? I think it could be possible that the container has different naming/number count on other computers depending on how docker compose created it in the first place (e.g. naming conflict). Could we achieve the same with a docker compose run command?

Reply:
Good point! Changed it to docker compose exec.